docs/source/execution_device_interoperability.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
<p>I think we should encourage users to use strings with the global device type utility. From an UX perspective, only needing to think about one function feels cleaner and with less indirection.</p><p> </p><p>I.e., "set device as cpu or gpu" rather than "set device as devicetype.host or devicetype.device" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/5130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>